src/ISA/ISA.Spreadsheet/CompositeCell.fs

```diff
@@ -0,0 +1,24 @@
+﻿module ISA.Spreadsheet.CompositeCell
+
+open ISA
+open FsSpreadsheet
+
+let fromFsCells (cells : list<FsCell>) : CompositeCell =
```

Comment:
This is meant to be used on grouped cells? This should be written as `<summary>`. Also implement descriptive failwith message.